test/src/plugins/array.spec.js

```diff
@@ -17,13 +17,13 @@ describe('plugins/array', () => {
   })
   describe('create', () => {
     it('returns a new instance of the array with mods applied', () => {
-      const original = [ 'foo', 'bar' ]
+      const original = [ 'foo', { fizz: 'buzz' }, 'bar' ]
       const actual = array.create(original, {
         add: [ 'baz' ],
         remove: [ 'bar' ]
       })
-      expect(original).to.deep.equal([ 'foo', 'bar' ])
-      expect(actual).to.deep.equal([ 'foo', 'baz' ])
+      expect(original).to.deep.equal([ 'foo', { fizz: 'buzz' }, 'bar' ])
```

Comment:
May want to perform two comparisons here, one to check that references are broken, and the other to confirm deep equality:

```js
const obj = { fizz: 'buzz' }
const original = [ 'foo', { fizz: 'buzz' }, 'bar' ]

// compare references
expect(original[1]).to.not.equal(actual[1])

// compare equality
expect(original[1]).to.deep.equal(actual[1])
```

This way you're ensuring the feature fundamentally offered by `fixd`, namely that objects cannot be mutated between tests.